dxlclient/client.py

```diff
@@ -1255,3 +1235,51 @@ def unregister_service_sync(self, service_req_info, timeout):
 
             self._service_manager.remove_service(service_req_info.service_id)
             service_req_info._wait_for_unregistration(timeout=timeout)
+
+    def _get_mqtt_client(self):
+        """
+        Returns the mqtt client instance based on if web socket proxy is enabled or not
+        :return: MQTT client instance
+        """
+        if self._proxy is not None:
+            mqtt_client = mqtt_dxl
+        else:
+            mqtt_client = mqtt
+
+        return mqtt_client.Client(client_id=self._config._client_id,
+                                  clean_session=True,
+                                  userdata=self,
+                                  protocol=mqtt.MQTTv31,
```

Comment:
In Java we switched this to use MQTTv311 for the protocol. Not sure if we want to do that here too.